src/http/lib/pubsub.ts

```diff
@@ -23,30 +23,34 @@ export default class DaprPubSub {
     return res.status;
   }
 
-  async subscribe(pubSubName: string, topic: string, cb: TypeDaprPubSub) {
+  async subscribe(subs: TypeElementOfDaprPubSub[]) {
     const server = await WebServerSingleton.getServer();
+    if (subs.length < 1) return;
+    
+    subs.forEach((sub: TypeElementOfDaprPubSub, i) => {
```

Comment:
Would love a Promise.all here, please do a .map and apply the .map to a Promise.all for parallel registration